mitmproxy/utils/human.py

```diff
@@ -62,3 +62,10 @@ def format_timestamp(s):
 def format_timestamp_with_milli(s):
     d = datetime.datetime.fromtimestamp(s)
     return d.strftime("%Y-%m-%d %H:%M:%S.%f")[:-3]
+
+
+def format_address(s):
+    # s is the address tuple
```

Comment:
This function should have a proper docstring explaining that it accepts hostnames and IPv4/IPv6 address tuples. It may also be useful to add type annotations to make this clear.